dotnet-projects/StyleCop/StyleCop/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace StyleCop.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
```

Comment:
Let's break this into multiple lines. 
please change the snippet in the article as well. thanks